### DISCUSSION TITLE
[Script request]

### DISCUSSION THREAD

**Discussion by rocko-gft** [community-scripts/ProxmoxVE#1053] [Request script]
### Application Name

Immich

### Website

https://immich.app/

### Description

Immich is the self-hosted version of Google Photos, which does not require you to share all your photos. 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by oOStroudyOo** [comment]
Hello,

This is a duplicate of #43 
Please see the resolution of this request.

Thank you.
